data/inventory.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
I doubled the number of cars, then added the id of the odd rows to the links in the nav